lib/ai_bot/tools/random_picker.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+module DiscourseAi
+  module AiBot
+    module Tools
+      class RandomPicker < Tool
+        def self.signature
+          {
+            name: name,
+            description:
+              "Handles a variety of random decisions based on the format of each input element",
+            parameters: [
+              {
+                name: "options",
+                description:
+                  "An array where each element is either a range (e.g., '1-6') or a comma-separated list of options (e.g., 'sam,jane,joe')",
+                type: "array",
+                item_type: "string",
+                required: true,
+              },
+            ],
+          }
+        end
+
+        def self.name
```

Comment:
That's just me, but I think it's always risky to override name in ruby. Given name is defined on modules. It can lead to oddities, probably fine here.

```
module Test
end

Test.name # "Test"
```

Reply:
hmm interesting, will be quite an involved change cause we would need to change all tools, and then have lack of parity with #description ... will have a think.